chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -96,6 +96,9 @@ menu-view-columns-move-left =
 menu-view-columns-move-right =
     .label = Move Column Right
 
+menu-edit-copy-annotation =
+    .label = Copy Annotation
```

Comment:
When multiple annotations are selected, this should say "Copy [n] Annotations", using `$count`.

Reply:
(We don't actually do this properly for "Copy Citation", but we might as well get this one right.)

Reply:
(Or it could be "Copy Annotations", but that's probably not any easier. Finder does "Copy 5 Items" for multiple-selection.)

Reply:
Totally! The "Copy Citation" is actually what threw me off here - I thought maybe the annotations menu should be consistent with it. But you're right, might as well make this one right. So it says "Copy [n] Annotations" for many annotation rows now.